### ISSUE TITLE
[Accessibility] TextInput double tap does not work after hitting back button and going back to Textinput, when Talkback is on

### ISSUE THREAD

**Issue by sugandhi-ms** [facebook/react-native#31395]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
With Talkback on, activate TextInput by double tapping. Start typing, then hit back button on keyboard. Now focus TextInput again and double tap. Keyboard does not show up.

## React Native version:
0.62.2

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1.Activate TextInput by double tapping. Start typing, then hit back button on keyboard. Now focus TextInput again and double tap. Keyboard does not show up.

## Expected Results
Double tap on TextInput should open keyboard.

**Comment by alextait1** [comment]
We'd like to reproduce this, can you send a code sample with more specifics?
